kolibri/plugins/coach/assets/src/views/plan/PlanQuizPreviewPage.vue

```diff
@@ -55,11 +58,15 @@
       ...mapActions('examCreation', ['addToSelectedExercises', 'removeFromSelectedExercises']),
       handleAddResource(content) {
         this.addToSelectedExercises([content]);
-        this.createSnackbar(this.$tr('added', { item: this.currentContentNode.title }));
+        this.$router.push(this.returnBackRoute).then(() => {
+          this.createSnackbar(this.$tr('added', { item: this.currentContentNode.title }));
```

Comment:
You can fix #7212 at this line by adding the `commonCoreString` mixin (which provides a `showSnackbarNotification` method) to the component and replacing the line here

```suggestion
          this.showSnackbarNotification('resourcesAddedWithCount', { count: 1 });
```